wrapper/tests/test_v2v_args.py

```diff
@@ -62,6 +73,8 @@ def test_vddk_basic(self):
             '-io', 'vddk-libdir=/opt/vmware-vix-disklib-distrib',
             '-io', 'vddk-thumbprint=01:23:45:67:89:AB:CD:EA:DB:EE:F0:12:34:56:78:9A:BC:DE:F0:12',  # NOQA E501
             '--password-file', '/vmware/password',
+            '--key', '/dev/sda1:file:/tmp/luks/sda1',
+            '--key', '/dev/sda2:file:/tmp/luks/sda2',
```

Comment:
create new test function for the LUKS related changes instead of reusing `test_vddk_basic()`